core/src/test/scala/org/broadinstitute/dsde/rawls/entities/EntityServiceSpec.scala

```diff
@@ -189,6 +189,23 @@ class EntityServiceSpec extends AnyFlatSpec with ScalatestRouteTest with Matcher
     }
   }
 
+  it should "fail to rename an entity type to a name already in use"  in withTestDataServices { services =>
+    val waitDuration = Duration(10, SECONDS)
+    val ex = intercept[RawlsExceptionWithErrorReport] {
+      Await.result(services.entityService.renameEntityType(testData.wsName, testData.pair1.entityType, testData.pair1.entityType), waitDuration)
+    }
+    ex.errorReport.message shouldBe "Pair already exists as an entity type"
+  }
+
+  it should "rename an entity type as long as the selected name is not in use"  in withTestDataServices { services =>
+    val waitDuration = Duration(10, SECONDS)
+    assertResult(1) {Await.result(services.entityService.renameEntityType(testData.wsName, testData.pair1.entityType, "newPair"), waitDuration)}
+    // verify there are no longer any entities under the old entity name
+    val queryResult = Await.result(services.entityService.listEntities(testData.wsName, testData.pair1.entityType), waitDuration)
+    assert(queryResult.isEmpty)
+  }
```

Comment:
add a test for an attempt to rename an entity type that doesn't exist (i.e. "oldName" is incorrect")

Reply:
Done